frontend/src/components/cronjob/Details.tsx

```diff
@@ -54,6 +71,103 @@ export default function CronJobDetails() {
 
   const ownedJobs = filterOwnedJobs(jobs);
 
+  // method to generate a unique string
+  const uniqueString = () => {
+    const timestamp = Date.now().toString(36);
+    const randomNum = Math.random().toString(36).substr(2, 5);
+    return `${timestamp}-${randomNum}`;
+  };
+
+  function SpawnJobDialog() {
+    const job = _.cloneDeep(cronJob.spec.jobTemplate);
+    const [jobName, setJobName] = useState(
+      `${cronJob?.metadata?.name}-${'manual-job-'}${uniqueString()}`
```

Comment:
You don't need this `${'manual-job-'}` as a variable.
Nit: Maybe `-manual-spawn-` makes more sense.

Reply:
updated